### PR TITLE
853 recreate boxes view

### PR DESCRIPTION
**Pull request by flisowna**
This is part 1 of the implementation of the boxes view in the new frontend. Follow ups to do already identified (there might be more): 
1. adding tests
2. as soon as the feature "base selector for the current session" is implemented: change the currently hard coded baseId (1) to the selected baseId (@spaudanjo)
3. write an ADR proposal statement for using chakra-ui and react-table (@spaudanjo)
4. optimise for mobile screens

Contributes to https://trello.com/c/BGjslF4u/853-recreate-boxes-list-view-in-new-fe

### CODE REVIEW

**Review by vahidbazzaz — APPROVED** [review]
thanks @flisowna

**Comment by spaudanjo** [comment]
Thanks @vahidbazzaz :) 
@flisowna will merge in a few hours as soon as she is back

File: react/src/types/react-table-config.d.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import {
```
Comment (spaudanjo) [inline react/src/types/react-table-config.d.ts:1]: Maybe a comment with a "Why is this needed" and a link (SO, github issue to the react-table project etc) could help here?

**Review by spaudanjo — APPROVED** [review]
I left one code comment - if you don't think it's needed, feel free to merge (but I think it would be helpful to add this info to the react-table types file) :)

File: react/src/types/react-table-config.d.ts (hunk 1 above)
Comment (flisowna) [reply react/src/types/react-table-config.d.ts:3]: I added a comment :)
